latex.ddoc

```diff
@@ -260,6 +260,7 @@ SINGLEQUOTE=`$0'
 SMALL=\small{$0}
 SPEC_S=$(LAYOUT , $1, $(ARGS $+))
 SUB=\textsubscript{$0}
+SUBSCRIPT=\textsubscript{$0}
```

Comment:
This is exactly similar to the existing, unused `SUB` macro. Which one do you prefer? (let's remove the other)

Reply:
SUB is already "subset", so it should be deleted. I'll do this in a moment.